client/src/modules/dashboard/Events/components/SendEmailModal.tsx

```diff
@@ -0,0 +1,109 @@
+import { Button } from '@chakra-ui/button';
+import { Checkbox } from '@chakra-ui/checkbox';
+import { Stack } from '@chakra-ui/layout';
+import {
+  Modal,
+  ModalBody,
+  ModalContent,
+  ModalHeader,
+  ModalCloseButton,
+  ModalFooter,
+  ModalOverlay,
+} from '@chakra-ui/modal';
+import { Alert, AlertIcon, AlertDescription } from '@chakra-ui/react';
+import React from 'react';
+import { useForm } from 'react-hook-form';
+import { useSendEventInviteMutation } from 'generated/graphql';
+interface SendEmailModalProps {
+  onClose: () => any;
+  isOpen: boolean;
+  eventId: number;
+}
+const SendEmailModal: React.FC<SendEmailModalProps> = ({
+  onClose,
+  isOpen,
+  eventId,
+}) => {
+  const {
+    register,
+    getValues,
+    handleSubmit,
+    formState: { errors },
+  } = useForm();
```

Comment:
```suggestion
  } = useForm<FormInputs>({ defaultValues: { confirmed: true } });
```
The `FormInput` bit restricts the type of values the form can handle.  More on that below! Also, if we use `defaultValues` rather than `defaultChecked` the form will remember the current form state when we close and reopen it.

Reply:
Updated the code to use FormInput type and used default values in useForm.